Source/Extensions/Blazorise.Charts/ChartColor.cs

```diff
@@ -40,8 +41,123 @@ public ChartColor( float red, float green, float blue, float alpha )
 
     #endregion
 
+    #region Operators
+
+    /// <summary>
+    /// Implicitly converts from string value to <see cref="ChartColor"/>.
+    /// </summary>
+    /// <param name="value">String value.</param>
+    public static implicit operator ChartColor( string value )
```

Comment:
Does not support, "red, blue,etc..." I'm guessing?

Reply:
Nop.